src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -315,15 +342,18 @@ export const GetFormHelp = () => {
   );
 };
 
+
 export const ITFDescription = (
   <span><strong>Note:</strong> By clicking the button to start the disability application, you’ll declare your intent to file, and this will set the date you can start getting benefits. This intent to file will expire 1 year from the day you start your application.</span>
 );
 
+
```

Comment:
Are these lines accidental?

Reply:
No. I just like two spaces between exports, so I figured while I was already there, I may as well take 10 seconds to add them. :man_shrugging: